credentials.go

```diff
@@ -25,6 +25,7 @@ import (
 
 var credentialLocations = []string{
 	"/etc/zypp/credentials.d/SCCcredentials",
+	"/run/secrets/credentials.d/SCCcredentials",
```

Comment:
Right now the docker patch puts it into /run/secrets/SCCcredentials.

Either you use that path or we need to change the Docker one.